Frontend/src/components/SignIn.jsx

```diff
@@ -1,9 +1,252 @@
+                                            borderRadius="4px"
+                                            bgColor="#0F0F0F"
+                                            color="#FFFFFF"
+                                            className="small"
+                                            fontWeight="bold"
+                                            border="1px solid #FFFFFF"
+                                            boxShadow="-2px 4px 6px rgba(0, 0, 0, 0.5)"
+                                            _hover={{
+                                                cursor: "pointer",
+                                                color: "#E5FFE4",
+                                            }}
+                                            _active={{
+                                                boxShadow:
+                                                    "-0px 2px 6px rgba(0, 0, 0, 0.5)",
+                                                transform:
+                                                    "translateY(1px) translateX(-1px)",
+                                            }}
+                                        >
+                                            SignIn
+                                        </Button>
+                                    </Flex>
+                                </Form>
+                            )}
+                        </Formik>
+                        <Flex
+                            height="200px"
+                            width="100%"
+                            flexDirection="column"
+                            position="absolute"
+                            bottom="12px"
+                            justifyContent="space-around"
+                            alignItems="center"
+                        >
+                            <Flex
+                                flexDirection="row"
+                                justifyContent="center"
+                                alignItems="center"
+                            >
+                                <Box
+                                    height="1px"
+                                    width="200px"
+                                    backgroundColor="#4A4A4A"
+                                ></Box>
+                                <Text
+                                    margin="8px"
+                                    fontWeight="bold"
+                                    className="x-small"
+                                    color="#4A4A4A"
+                                >
+                                    or
+                                </Text>
+                                <Box
+                                    height="1px"
+                                    width="200px"
+                                    backgroundColor="#4A4A4A"
+                                ></Box>
+                            </Flex>
+                            {<GoogleButton />}
```

Comment:
Why have you kept `<GoogleButton />` in {} here?

Reply:
Hey Neel, it was first commented on. When uncommenting I forgot to remove the curly brackets and removed only the asterisks and slashes. It doesn't affect the code but it will look inconsistent. I'll remove that rn.